### ISSUE TITLE
require 'app_store_connect' is not working

### ISSUE THREAD

**Issue by jonatasteixeira** [kyledecot/app_store_connect#96]
**Describe the bug**
When I'm getting a exception when I try to require

**To Reproduce**
Run the follow:
`require 'app_store_connect'`

**Expected behavior**

```
irb(main):008:0> require "app_store_connect"
Traceback (most recent call last):
       16: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/app_store_connect-0.20.0/lib/app_store_connect/client.rb:3:in `<top (required)>'
       15: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require'
       14: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require'
       13: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/activesupport-4.2.11.1/lib/active_support/all.rb:3:in `<top (required)>'
       12: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require'
       11: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require'
       10: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/activesupport-4.2.11.1/lib/active_support/core_ext.rb:1:in `<top (required)>'
        9: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/activesupport-4.2.11.1/lib/active_support/core_ext.rb:1:in `each'
        8: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/activesupport-4.2.11.1/lib/active_support/core_ext.rb:2:in `block in <top (required)>'
        7: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
        6: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
        5: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/activesupport-4.2.11.1/lib/active_support/core_ext/object.rb:3:in `<top (required)>'
        4: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require'
        3: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require'
        2: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/activesupport-4.2.11.1/lib/active_support/core_ext/object/duplicable.rb:106:in `<top (required)>'
        1: from /Users/jonatasteixeira/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/activesupport-4.2.11.1/lib/active_support/core_ext/object/duplicable.rb:111:in `<class:BigDecimal>'
NoMethodError (undefined method `new' for BigDecimal:Class)
````

**Environment (please complete the following information):**
 - OS: MacOS 10.14.6
 - Ruby 2.7.1
 - Rbenv 1.1.2

**Comment by kyledecot** [comment]
Hi @jonatasteixeira 👋 What version of the gem are you using and how did you install it (`Gemfile` or `bundle install app_store_connect`)?

**Comment by kyledecot** [comment]
I just released `0.21.0` which should this issue but will require a later version of active support. See https://github.com/kyledecot/app_store_connect/pull/97 for more details.
